libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/ui/SalesforceDroidGapActivity.java

```diff
@@ -394,7 +406,7 @@ public void run() {
                         SalesforceDroidGapActivity.this.client = SalesforceDroidGapActivity.this.clientManager.peekRestClient();
                         setSidCookies();
                         loadVFPingPage();
-                        final String frontDoorUrl = getFrontDoorUrl(url, true);
+                        final String frontDoorUrl = getFrontDoorUrl(url, BootConfig.isAbsoluteUrl(url));
```

Comment:
This shouldn't be always `true`. This parameter specifies if the passed in URL is an absolute URL or not.